app/javascript/modules/listingDetails/ListingDetailsPricingTable.tsx

```diff
@@ -61,11 +61,11 @@ const buildSaleHoaDuesCellRow = (unit: RailsUnitWithOccupancyAndMaxIncome) => {
   if (unit?.HOA_Dues_With_Parking && unit?.HOA_Dues_Without_Parking) {
     return [
       {
-        cellText: `$${(unit?.HOA_Dues_With_Parking).toLocaleString()}`,
+        cellText: `$${Math.round(unit?.HOA_Dues_With_Parking).toLocaleString()}`,
```

Comment:
I thought these were removed so that we could keep the cents as part of [DAH-1085](https://sfgovdt.jira.com/browse/DAH-1085). Is that no longer the case?

Reply:
ah! My bad, I'm very grateful you caught it. Thanks * a million. This is from a mistaken merge conflict resolution. I'll fix.